.github/workflows/prerelease_linux.yml

```diff
@@ -0,0 +1,114 @@
+name: Prerelease Linux pipeline
+
+on:
+  release:
+    types:
+      - prereleased
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GO_VERSION: '1.14.4'
+  GPG_MAIL: 'infrastructure-eng@newrelic.com'
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  SNYK_TOKEN: ${{ secrets.COREINT_SNYK_TOKEN }}
+  TAG: ${{ github.event.release.tag_name }}
+  DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+  DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+
+jobs:
+  unit-test-linux:
+    name: Linux unit tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{env.GO_VERSION}}
+
+      - name: Validate code
+        run: make ci/validate
+
+      - name: Running unit tests
+        run: make ci/test-coverage
+
+      - name: Install gcov2lcov
+        working-directory: /tmp
+        run: go get -u github.com/jandelgado/gcov2lcov
+
+      - name: Convert coverage.out to lcov.info
+        run: gcov2lcov -infile=coverage.out -outfile=lcov.info
+
+      - name: Coveralls Parallel
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
+          path-to-lcov: lcov.info
+          flag-name: run-linux
+          parallel: true
+
+  harvest-test-linux:
+    name: Linux harvest tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, ubuntu-16.04 ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        run: sudo apt install musl-dev
```

Comment:
can we add a comment explaining why this is required?

Reply:
I've removed this dependency as wasn't required anymore. Thanks!